packages/components/package.json

```diff
@@ -35,6 +37,7 @@
 		"classnames": "^2.2.5",
 		"clipboard": "^2.0.1",
 		"dom-scroll-into-view": "^1.2.1",
+		"hex-rgb": "^4.1.0",
```

Comment:
`hex-rgb` uses modern JavaScript (`const`, `let`, etc.) and is not transpiled to be able to run in older browsers.

tl;dr: Including this library has broken the editor in Internet Explorer.

Apparently this is an intentional choice on their part: https://github.com/sindresorhus/hex-rgb/issues/10#issuecomment-419901794

> This module mainly targets Node.js, not the browser. It's up to you to transpile it with Babel if you want to use it in the browser.

If it's something we plan to use, it would need to be transpiled as well.  I don't recall having had to deal with this for any other dependencies, and it would probably require some work to adapt into our [build tooling](https://github.com/WordPress/gutenberg/blob/master/bin/packages/build.js).

Reply:
Good catch! Long term I think its probably good idea to support this in our tooling just as [CRA](https://github.com/facebook/create-react-app/issues/1125#issuecomment-426120762), Parcel and other tools in the JavaScript ecosystem have done. Switching to [`polished`](https://www.npmjs.com/package/polished) [`parseToRgb`](https://polished.js.org/docs/#parsetorgb) or similar fn might be a quick fix.

Reply:
BTW thinking about the longer-term fix, I saw this on Twitter the other day and it seems that not compiling (or only compiling so-far) is closer to becoming a defacto standard (yay for smaller bundle sizes apps supporting modern browsers!).

https://twitter.com/_developit/status/1196393947989512193

https://twitter.com/_developit/status/1196393947989512193

Reply:
PR for updating the build tooling https://github.com/WordPress/gutenberg/pull/18798